packages/js/src/ui/components/Renderer.tsx

```diff
@@ -65,6 +65,7 @@ type RendererProps = {
   options: NovuOptions;
   tabs: Array<Tab>;
   preferencesFilter?: PreferencesFilter;
+  preferenceGroups?: PreferenceGroups;
```

Comment:
the new Inbox prop according to [the DX guide](https://www.notion.so/novuhq/Preference-Grouping-1edd03ad838f807db7fcde919aa897b9?pvs=4#1edd03ad838f807f92f4d7c0af0370c8)